### ISSUE TITLE
Need to change colors of website

### ISSUE THREAD

**Issue by lakshyatyagi24** [lakshyatyagi24/Furniture-Website#1]
Colors of website is not looking classic. Need to improve color combination

**Comment by Sandradell** [comment]
Please, assign me

**Comment by lakshyatyagi24** [comment]
Sure you can fork this repo, give a star and start working on it.

After making a pull request i will review your work and merge accordingy

**Comment by ShobhitRathi** [comment]
i am good with color schemes as well if you want i can change it and make your website look professional. already di your navbar as well

**Comment by lakshyatyagi24** [comment]
sure you can

**Comment by tahseenzahra** [comment]
i have seen your repo and was working on it from the last couple of hours and selected the colour scheme  can I work on it

**Comment by lakshyatyagi24** [comment]
before starting can you please share me color scheme

**Comment by tahseenzahra** [comment]
means before creating pull request

**Comment by lakshyatyagi24** [comment]
just tell me color code

**Comment by tahseenzahra** [comment]
for body bg i choose #f8f8ff
for footer and header, I selected grey and the area where pics were there I choose as rgb(40,40,40).
i completed all work too

**Comment by prabhu4444** [comment]
Is this issue still open?

**Comment by prabhu4444** [comment]
> before starting can you please share me color scheme

Your current scheme is too dark, it's not giving it a formal official look.

**Comment by prabhu4444** [comment]
Suggestion: Lighter colors give a formal look

Do you want a lighter color or you want to stick with your present choice of color?
Tell us what you want so everyone can work accordingly.

**Comment by lakshyatyagi24** [comment]
Yes @prabhu4444 it is still open

**Comment by tahseenzahra** [comment]
yup depends on colour choice see if there is can't find any other issue in mine you can merge it and it would be easy for @prabhu4444 to change colours from .css files as I made classes, with a thought that it might be useful for you to change colour just at one place in future if you need because these colours were changing on every page separately
